x-pack/platform/plugins/private/upgrade_assistant/public/application/components/overview/ml_callout/ml_callout.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const MachineLearningDisabledCallout: React.FunctionComponent = () => {
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.upgradeAssistant.overview.mlCallout.title"
+            defaultMessage="Machine Learning is disabled"
```

Comment:
To avoid repetition, how about we remove the first sentence in the message and add it to the title instead? 
```suggestion
            defaultMessage="Machine Learning is disabled in your cluster"
```

Reply:
Sounds good to me! Adding the commit.